Kitodo/src/main/java/de/sub/goobi/helper/tasks/EmptyTask.java

```diff
@@ -58,20 +58,10 @@
      * which will automatically be attached to all task threads. Otherwise
      * exceptions might get lost or even bring the runtime to crash.
      */
-    public static final Thread.UncaughtExceptionHandler CATCH_ALL = new Thread.UncaughtExceptionHandler() {
-        /**
-         * The function uncaughtException() will catch any uncaught exception
-         * that might occur in the task and will store it in the
-         *
-         * @see java.lang.Thread.UncaughtExceptionHandler#uncaughtException(java.lang.Thread,
-         *      java.lang.Throwable)
-         */
-        @Override
-        public void uncaughtException(Thread origin, Throwable exception) {
-            if (origin instanceof EmptyTask) {
-                EmptyTask task = (EmptyTask) origin;
-                task.setException(exception);
-            }
+    public static final Thread.UncaughtExceptionHandler CATCH_ALL = (origin, exception) -> {
```

Comment:
JavaDoc did not matched or move this code into its own class to get ride out of this inner class usage.

Reply:
JavaDocs can't be made for lambdas - param attributes are not allowed there - and it is constant.